src/main/cpp/t06_cinema.cpp

```diff
@@ -29,6 +29,46 @@
 using namespace std;
 
 int t06_cinema() {
-    
+    short n = 0, m = 0;
+    cin >> n >> m;
+    const short visota = n;
+    const short shirina = m;
+    bool mass[visota][shirina];
+    for (int i = 0; i < visota; i++)
+    {
+        for (int j = 0; j < shirina; j++)
+        {
+            bool zn = 0;
+            cin >> zn;
+            mass[i][j] = zn;
+        }
+    }
+    short iskomiy_ryad = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'iskomiy_ryad' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4512898)